README.md

```diff
@@ -16,31 +16,100 @@ freeCodeCamp is a friendly open source community where you learn to code and hel
+**Add links to the live demos of your projects as you finish them.**
+a. Visualize Data With A Bar Chart
+b. Visualize Data With A Scatterplot Graph
+c. Visualize Data With A Heat Map
+d. Visualize Data With A Choropleth Map
+e. Visualize Data With A Treemap Diagram
+
+##### 5. APIs and Microservices
+
+**Add links to the live demos of your projects as you finish them.**
+a. Timestamp Microservice
+b. Request Header Parser Microservice
+c. URL Shortener Microservice
+d. Exercise Tracker
+e. File Metadata Microservice
+
+##### 6. Information Security and Quality Assurance
+
+**Add links to the live demos of your projects as you finish them.**
+a. Metric Imperial Converter
+b. Issue Tracker
+c. Personal Library
+d. Stock Price Checker
+e. Anonymous Message Board
+
+**Once you earn all 6 of these certificates, we’ll award you with the freeCodeCamp Full Stack Development Certificate. This final distinction signifies that you’ve completed around 1,800 hours of coding with a wide range of web development tools.**
+
+
+###These six new certifications will replace the old legacy certificates:
+
+**Legacy Front End Development**
+**Legacy Data Visualization**
+**Legacy Back End Development**
+
+###Coding for nonprofits
+
+We’ve decided to make coding for nonprofits a separate process from the certificate program. We’ve added 5 of our projects that are used by nonprofits - our Open Source for Good projects - to the challenge map. Feel free to browse their open issues. You can dive in and start contributing whenever you feel ready.
+
+We will continue to add Open Source for Good projects to the challenge map as they become stable and ready for your contributions. These projects are all built using the same JavaScript-based tools you’ll learn by earning the certificates. And several of these are nearing a public beta (I’ll formally announce these as they go into open beta).
+
+In addition to our Open Source for Good projects - which are particularly well-documented and beginner-friendly - there are tons of other open source projects you can contribute to. The community has built a guide to contributing to various open source projects here 363.
+
+###What happens to the old legacy certificates?
+
+If you’ve already earned a certificate, you will always have it. You will always be able to link to it from your LinkedIn or résumé. And when your prospective employers or freelance clients click that link, they’ll see a verified certificate specific to you.
+
+And if you’ve already started working toward the Legacy Front End Development, Legacy Data Visualization, or Legacy Back End Development certificates, and still want to claim it, don’t worry - you can will always be able to claim these certificates - even after we release the beta challenges.
+
+All of the required projects for these old certificates will remain available on freeCodeCamp. For the old certificates, you can complete old versions of the projects.
```

Comment:
There are no 'old versions' of the projects. For example, the Tribute Page project is the Tribute Page project for both current and legacy certifications.